### DISCUSSION TITLE
docs.github.com doesn't honor the 'Dark high contrast' theme preference

### DISCUSSION THREAD

**Discussion by daizuke123** [community/community#69059] [Accessibility] [answered]
[docs.github.com](https://docs.github.com/) doesn't honor the 'Dark high contrast' theme preference and falls back to the 'Default light' theme

**Comment by daizuke123** [comment, marked as answer]
Sometimes, cached data in your browser can cause display issues. Clear your browser's cache and cookies, then try enabling the "Dark high contrast" theme again.

**Reply by daizuke123** [reply]
Browser extensions or add-ons can sometimes interfere with website functionality. Disable any extensions that might affect the appearance or behavior of websites and see if the "Dark high contrast" theme works as expected.

**Comment by cehfisher** [comment]
Hi @daizuke123 - this theme preference issue appears to be resolved now. Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)
